app/views.py

```diff
@@ -0,0 +1,214 @@
+from django.core.mail import EmailMessage, send_mail
+from django.contrib import messages
+from django.shortcuts import render, get_object_or_404, redirect
+from django.contrib.auth import authenticate, login, logout, get_user_model
+from django.contrib.auth.decorators import login_required
+from django.utils import timezone
+
+from .models import *
+from django import forms
+
+
+
```

Comment:
please use class based views instead of fucntion based, and before pushing changes make sure to remove any debug/print statements